tests/test_integration.py

```diff
@@ -179,10 +183,9 @@ def test_remind_at(irc, user):
 
     reminder = irc.bot.memory[MEMORY_KEY][0]
     when = datetime.fromtimestamp(reminder.timestamp, pytz.utc)
-
```

Comment:
Line note just to point out that this blank line was not removed from the earlier test case. IMO it makes the test a bit easier to read.